host-configs/docker/docker-linux-ubuntu16.04-ivybridge-gcc@8.1.0.cmake

```diff
@@ -0,0 +1,82 @@
+#------------------------------------------------------------------------------
+# !!!! This is a generated file, edit at own risk !!!!
+#------------------------------------------------------------------------------
+# SYS_TYPE: linux-ubuntu16.04-ivybridge
+# Compiler Spec: gcc@8.1.0
+#------------------------------------------------------------------------------
+# CMake executable path: /usr/bin/cmake
+#------------------------------------------------------------------------------
+
+#------------------------------------------------------------------------------
+# Compilers
+#------------------------------------------------------------------------------
+
+set(CMAKE_C_COMPILER "/usr/bin/gcc" CACHE PATH "")
+
+set(CMAKE_CXX_COMPILER "/usr/bin/g++" CACHE PATH "")
+
+set(ENABLE_FORTRAN ON CACHE BOOL "")
+
+set(CMAKE_Fortran_COMPILER "/usr/bin/gfortran" CACHE PATH "")
+
+set(CMAKE_C_FLAGS "-pthread" CACHE PATH "")
+
+set(CMAKE_CXX_FLAGS "-pthread" CACHE PATH "")
+
+set(BLT_CXX_STD "c++14" CACHE PATH "")
+
+#------------------------------------------------------------------------------
+# TPLs
+#------------------------------------------------------------------------------
+
+# Root directory for generated TPLs
+set(TPL_ROOT "/home/axom/serac_tpls/gcc-8.1.0" CACHE PATH "")
```

Comment:
The new host-config here:

https://github.com/LLNL/axom/suites/1606131918/artifacts/29349708

doesn't have serac_tpls in the path.  It should fix the CMake error in the gcc@8 checks.